src/main/java/liquibase/ext/spanner/CloudSpannerConnection.java

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package liquibase.ext.spanner;
+
+import com.google.cloud.spanner.jdbc.JdbcDriver;
+import java.sql.Connection;
+import java.sql.Driver;
+import java.util.Properties;
+import liquibase.database.DatabaseConnection;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.DatabaseException;
+import liquibase.servicelocator.LiquibaseService;
+
+@LiquibaseService()
```

Comment:
Do we need LiquibaseService here? And, if so, would we need LiquibaseService in all of our other services?

Reply:
No, that is apparently not needed (copied it from other custom Connection implementations, but that is probably an old requirement that is no longer needed).